pepper-apis/configure.rb

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/ruby -w
 # Docker Image used to call out to generate config files
-$dsde_toolbox_image_name = "broadinstitute/dsde-toolbox:latest"
-
+$dsde_toolbox_image_name = "broadinstitute/study-server-build:java-2022-06-01A"
```

Comment:
This `configure.rb` script actually calls `consult-template` for us using Docker. Updating to use one of our new images (both of our build images have `consul-template` but chose this one because it is smaller and presumably will be quicker to download.